adafruit_button.py

```diff
@@ -38,21 +38,50 @@
 
 """
 
+from micropython import const
 import displayio
 from adafruit_display_text.text_area import TextArea
-from adafruit_bitmap_font import bitmap_font
 from adafruit_display_shapes.rect import Rect
 from adafruit_display_shapes.roundrect import RoundRect
 
 __version__ = "0.0.0-auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_Display_Button.git"
 
 
+def _check_color(color):
+    # if a tuple is supplied, convert it to a RGB number
+    if isinstance(color, tuple):
+        r, g, b = color
+        return int((r << 16) + (g << 8) + (b & 0xff))
+    return color
+
+
 class Button():
+    # pylint: disable=too-many-instance-attributes, too-many-locals
+    """Helper class for creating UI buttons for ``displayio``.
+
+    :param x: The x position of the button.
+    :param y: The y position of the button.
+    :param width: The width of the button in pixels.
+    :param height: The height of the button in pixels.
+    :param name: The name of the button.
+    :param style: The style of the button. Can be RECT, ROUNDRECT, SHADOWRECT, SHADOWROUNDRECT.
+                  Defaults to RECT.
+    :param fill_color: The color to fill the button. Defaults to 0xFFFFFF.
+    :param outline_color: The color of the outline of the button.
+    :param label: The button label. Defaults to 0x0.
```

Comment:
label is the button text that appears inside it, by default nothing is displayed